### PR TITLE
Fix for #18: Do not flag issues as bug

### PR DESCRIPTION
**Pull request by lmarniazman**
Newly created issues are now of type 'code smell' and tagged with 'architecture-violation'.

### CODE REVIEW

**Comment by lmarniazman** [comment]
For more information on the failed Travis CI build see #21. 
JDK installation seems to work when using 'openjdk8', like in PR #22.

File: src/main/java/com/devonfw/ide/sonarqube/common/impl/DevonSonarDefinition.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ private void addRule(NewRepository repository, Class<?> ruleClass) {
       tags[i] = tags[i].toLowerCase(Locale.US);
     }
     rule.setTags(tags);
-    rule.setType(RuleType.BUG);
+    rule.setType(RuleType.CODE_SMELL);
```
Comment (hohwille) [inline src/main/java/com/devonfw/ide/sonarqube/common/impl/DevonSonarDefinition.java]: perfect 👍

File: src/main/java/com/devonfw/ide/sonarqube/common/impl/check/DevonArchitecture3rdPartyHibernateCheck.java [hunk 2]
```diff
@@ -14,7 +14,7 @@
  */
 @Rule(key = "Devon4j:E4", name = "Devon 3rd Party Hibernate Check", //
     description = "Verify that Hibernate is properly used (and JPA is preferred).", //
-    priority = Priority.CRITICAL, tags = { "bug" })
+    priority = Priority.CRITICAL, tags = { "architecture-violation" })
```
Comment (hohwille) [inline src/main/java/com/devonfw/ide/sonarqube/common/impl/check/DevonArchitecture3rdPartyHibernateCheck.java]: excellent :)

**Comment by hohwille** [comment]
As the changes seem obvious and the travis issue is clarified, I merge this even with travis failure.

**Pull request merged**